zio-sbt-ci/src/main/scala/zio/sbt/ZioSbtCiPlugin.scala

```diff
@@ -111,7 +110,9 @@ object ZioSbtCiPlugin extends AutoPlugin {
               SetupLibuv,
               SetupJava(javaVersion),
               CacheDependencies
-            ) ++ checkAllCodeCompiles ++ checkArtifactBuildProcess ++ checkWebsiteBuildProcess
+            ) ++ checkAllCodeCompiles.flatMap(_.flatten) ++ checkArtifactBuildProcess.flatMap(
+              _.flatten
+            ) ++ checkWebsiteBuildProcess.flatMap(_.flatten)
```

Comment:
Why this boilerplate change is needed? Can you keep it as simple as it is?

Reply:
yes, I made a custom codec for Seq[Step] in Job